src/dmi.rs

```diff
@@ -63,3 +67,13 @@ fn create_png(path: &str, width: &str, height: &str, data: &str) -> Result<()> {
     let mut writer = encoder.write_header()?;
     Ok(writer.write_image_data(&result)?)
 }
+
+fn resize_png(path: &str, width: &str, height: &str) -> Result<()> {
+    let width = u32::from_str_radix(width, 10)?;
+    let height = u32::from_str_radix(height, 10)?;
+
+    let img = image::open(path).unwrap();
```

Comment:
I would much rather return an error using `?` rather than panicking if loading or saving fails

Reply:
Doooes that just mean add a question mark to the end of this line? Could you make a code suggestion? I'm really unfamiliar with rust's syntax and error handling.